framework/include/kernels/WeirdTestKernel.h

```diff
@@ -0,0 +1,36 @@
+//* This file is part of the MOOSE framework
```

Comment:
shouldn't this be under test ?

Reply:
I don't see any reason why you would think that

Reply:
Maybe because it is a weird *test* kernel?